core/frontend/src/tile/map/MapLayerTileTreeReference.ts

```diff
@@ -13,22 +13,42 @@ import { IModelApp } from "../../IModelApp";
 import { IModelConnection } from "../../IModelConnection";
 import { createModelMapLayerTileTreeReference, MapLayerImageryProvider, TileTreeReference } from "../internal";
 
-/** @internal  */
+/**
+ * A reference to a [[TileTree]] to be used for map layer tiles.
+ * @beta
+ */
 export abstract class MapLayerTileTreeReference extends TileTreeReference {
+  /**
+   * Constructor for a MapLayerTileTreeReference.
+   * @param _layerSettings Map layer settings that are applied to the MapLayerTileTreeReference.
+   * @param _layerIndex The index of the associated map layer.
+   * @param iModel The iModel containing the MapLayerTileTreeReference.
+   */
   constructor(protected _layerSettings: MapLayerSettings, protected _layerIndex: number, public iModel: IModelConnection) {
```

Comment:
How does API tags deals with constructor shorthands?  For example, does iModel becomes automatically @beta ?
@pmconne I think we should avoid constructor shorthand?

Reply:
Changed to explicitly set the properties